project_event/models/project_task.py

```diff
@@ -396,20 +396,26 @@ def create_children_from_activity_create(self, children, parent_id):
 
     @api.multi
     def write(self, vals):
-        if self.is_activity():
-            return self.write_activity(vals)
-        else:
-            self.update_reservation_event(vals)
-            return super(Task, self).write(vals)
+        super(Task, self).write(vals)
```

Comment:
It will write 2 times?

Reply:
Remove the duplicate right  with update_reservation_events because values are set to vals in value return. I call super write() method one time . I also remove  super(Task, self).write(vals) from write_activity() method